source/data/signatures_schema.sql

```diff
@@ -49,5 +49,6 @@ CREATE TABLE signatures (
     from_one_click boolean,
     eu_data_processing_consent boolean,
     consent_content_version_id bigint,
-    daisy_chain_id_used bigint
-);
\ No newline at end of file
+    daisy_chain_id_used bigint,
```

Comment:
This is definitely out of scope for this PR, but we should make sure to only export FKs that refer to other tables that we also export

Reply:
Yeah... I feel like this update has revealed several columns we're exporting and probably shouldn't. Like attendee token.